portfolio/src/main/webapp/style.css

```diff
@@ -16,14 +17,97 @@
 
 .image-thumbnails img {
   margin: 10px;
-  max-height: 200px;
-  max-width: 200px;
+  max-height: 300px;
+  max-width: 300px;
 }
 
 .image-thumbnails img:hover {
   border: thick solid black;
 }
 
-#random-image-container img {
-  margin-top: 20px;
-}
\ No newline at end of file
+#random-caption-container {
+  text-align: center;
+  padding-left: 15px;
+  padding-right: 15px;
+}
+
+.row {
+  display: flex;
+}
+
+/* Create 2 equal columns that sits next to each other */
+.column {
+  flex: 50%;
+  padding: 15px;
+}
+
+/* Google Fonts */
+@import url(https://fonts.googleapis.com/css?family=Anonymous+Pro);
+@import url(https://fonts.googleapis.com/css?family=Lora);
+
+html{
+  min-height: 100%;
+  overflow: hidden;
+}
+
+body{
+  height: calc(100vh - 8em);
+  padding: 4em;
+  color: rgba(255,255,255,.75);
+  Font-Family: 'Anonymous Pro', Monospace;
+  Font-Size: 19px;
+  background-color: #013220; 
+}
+
+.line-1{
+    position: relative;
+    top: 50%;  
+    width: 24em;
+    margin: 0 auto;
+    text-align: center;
+    white-space: nowrap;
+    overflow: hidden;
+    transform: translateY(-50%);    
+}
+
+h1{
+  Font-Family: 'Lora';
+  Font-Size: 52px;
```

Comment:
nit: these should be lowercase

style guidance: https://google.github.io/styleguide/htmlcssguide.html#Capitalization